unime/vite.config.ts

```diff
@@ -44,5 +44,23 @@ export default defineConfig(async () => ({
   },
   optimizeDeps: {
     exclude: ['~icons/*'],
+    // #188: List of dependecies that Vite frequently optimizes.
+    // Use `include`, not `exclude`: https://github.com/sveltejs/kit/issues/11793#issuecomment-1965850225.
+    // TODO Check if this list can be removed after upgrading to Vite 5.
+    include: [
```

Comment:
I found another two that optimized on my machine:
```
'date-fns',
'date-fns/locale'
```

Reply:
Also this one:
```
@tauri-apps/plugin-barcode-scanner
```

Reply:
Added these dependencies.